berkeley-function-call-leaderboard/bfcl_eval/model_handler/model_style.py

```diff
@@ -15,3 +15,4 @@ class ModelStyle(Enum):
     COHERE = "cohere"
     WRITER = "writer"
     NOVITA_AI = "novita_ai"
+    Ling = "ling"
```

Comment:
It looks like you’re defining a new model style, but it’s not used anywhere. Specifically, your handler doesn’t call the `convert_to_tool` function at all ([code reference](https://github.com/ShishirPatil/gorilla/blob/main/berkeley-function-call-leaderboard/bfcl_eval/model_handler/utils.py#L68-L179)). If this isn’t needed, please remove the associated code to keep things tidy.